task/pytest/0.1/README.md

```diff
@@ -21,7 +21,7 @@ kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/py
 * **PYTHON**: The used Python version, more precisely the tag for the Python image (_default_: `latest`)
 * **ARGS**: The additional arguments to be used with pytest
 * **SOURCE_PATH**: The path to the source code (_default_: `.`)
-* **REQUIREMENTS_FILE**: The name of the requirements file inside the source location (_default_: `requirements.txt`)
+* **REQUIREMENTS_FILE**: The name of the requirements file inside the source location (_default_: `requirements.txt`), with fallback to the requirements file in the root location
```

Comment:
The _default_ should come at the end of the sentence rather than be included as a sub-clause of it.